manageiq-operator/pkg/helpers/miq-components/util.go

```diff
@@ -85,3 +91,68 @@ func addAnnotations(annotations map[string]string, meta *metav1.ObjectMeta) {
 		}
 	}
 }
+
+func InternalCertificatesSecret(cr *miqv1alpha1.ManageIQ, client client.Client) *corev1.Secret {
+	name := cr.Spec.InternalCertificatesSecret
+
+	secretKey := types.NamespacedName{Namespace: cr.Namespace, Name: name}
+	secret := &corev1.Secret{}
+	client.Get(context.TODO(), secretKey, secret)
+
+	return secret
+}
+
+func addInternalCertificate(cr *miqv1alpha1.ManageIQ, d *appsv1.Deployment, client client.Client, name string, mountPoint string) {
+	secret := InternalCertificatesSecret(cr, client)
+	if secret.Data[fmt.Sprintf("%s_crt", name)] != nil && secret.Data[fmt.Sprintf("%s_key", name)] != nil {
+		volumeName := fmt.Sprintf("internal-%s-certificate", name)
```

Comment:
what if the postgres is external - does this name then make it confusing?  is it better to just say "%s-certificate"?

Reply:
Done, btw... it's only used to link the volumeMount to the volumeSource, but I see the point.